features/javascript/README.md

```diff
@@ -42,7 +42,7 @@ const expectedResponse = {
   'body': '{\n  "origin": "94.113.241.2"\n}'
 };
 
-gavel.isValid(actualResponse, expectedResponse, 'response', (err, result) => {
+gavel.validate(actualResponse, expectedResponse, 'response', (err, result) => {
```

Comment:
here it's still async

Reply:
Yes, I'm not yet settled with the redesigned call signature of `validate`. Once done and discussed, I will update the behavior suites and examples 👍